### PR TITLE
feat: simplify workflows

### PR DESCRIPTION
**Pull request by segunadebayo**
**Description**

To help simplify and streamline our release and changelog process, we're adding Changeset to our toolchain.

Changeset helps in providing a detailed description of changes, versioning the packages, and releasing updates with proper changelogs generated. To learn more, check here: https://github.com/atlassian/changesets/tree/master/packages/cli

We ran into issues in the past using `lerna` and `lerna-changelog package`. Well, not really issues, but it was more of a crude process. We'll like to outsource this process to changeset to save us time. Time is precious ⏰

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,115 +46,37 @@
...
-    "select": "yarn workspace @chakra-ui/select",
-    "css-reset": "yarn workspace @chakra-ui/css-reset",
-    "media-query": "yarn workspace @chakra-ui/media-query",
-    "skeleton": "yarn workspace @chakra-ui/skeleton",
-    "skip-nav": "yarn workspace @chakra-ui/skip-nav",
-    "babel-plugin": "yarn workspace @chakra-ui/babel-plugin",
-    "theme-tools": "yarn workspace @chakra-ui/theme-tools",
-    "styled-system": "yarn workspace @chakra-ui/styled-system",
-    "control-box": "yarn workspace @chakra-ui/control-box",
-    "gatsby-plugin": "yarn workspace @chakra-ui/gatsby-plugin"
```
Comment (TimKolberger) [inline package.json:113]: Love this simplification ❤️

**Review by TimKolberger — APPROVED** [review]

File: .changeset/popular-foxes-behave.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+This uses `manypkg run` under the hood to run specific scripts in the workspace
+directories `packages`, `tooling`, etc.
+
+## Update Github workflow
+
+The publish workflow has been updated to include only `yarn release` in favor of
+adding a `prerelease` script to the `package.json`.
+
+The release script is not managed by `changeset` not `lerna`, so we run
```
Comment (with-heart) [inline .changeset/popular-foxes-behave.md:34]:
```suggestion
The release script is managed by `changeset`, not `lerna`, so we run
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -4,42 +4,22 @@ When it comes to open source, there are different ways you can contribute, all
...
-  - [Storybook](#storybook)
-- [Think you found a bug?](#think-you-found-a-bug)
-- [Proposing new or changed API?](#proposing-new-or-changed-api)
-- [Making a Pull Request?](#making-a-pull-request)
-  - [Commit Convention](#commit-convention)
-  - [Steps to PR](#steps-to-pr)
-  - [Tests](#tests)
-- [Want to write a blog post or tutorial](#want-to-write-a-blog-post-or-tutorial)
-- [Want to help improve the docs?](#want-to-help-improve-the-docs)
-- [License](#license)
```
Comment (with-heart) [inline CONTRIBUTING.md:22]: Do we really want to get rid of the ToC? Seems like it's potentially helpful

**Review by with-heart — APPROVED** [review]
Love it!

File: CONTRIBUTING.md (hunk 3 above)
Comment (segunadebayo) [reply CONTRIBUTING.md:22]: I feel it gets in the way of the most important information needed to contribute.

I've tried to onboard 2 friends and it initially felt like a lot of information just to contribute. I thought to limit that to just a few steps.

It's basically a test for "less is more". I can always regenerate it if we have a need for it.

Comment (with-heart) [reply CONTRIBUTING.md:22]: Yeah I fully agree with the "less is more" approach!

**Comment by ljosberinn** [comment]
looks good to me too, especially that huge package list removal 😆

**Review by ljosberinn — APPROVED** [review]

**Pull request merged**